src/Enterspeed.Source.UmbracoCms/Models/UmbracoMasterContentEntity.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using Enterspeed.Source.Sdk.Api.Models;
+using Enterspeed.Source.Sdk.Api.Models.Properties;
+using Enterspeed.Source.UmbracoCms.Extensions;
+using Enterspeed.Source.UmbracoCms.Services;
+using Umbraco.Cms.Core.Models.PublishedContent;
+
+namespace Enterspeed.Source.UmbracoCms.Models
+{
+    public class UmbracoMasterContentEntity : IEnterspeedEntity<IDictionary<string, IEnterspeedProperty>>
+    {
+        private readonly IPublishedContent _content;
+        private readonly IEntityIdentityService _entityIdentityService;
+
+        public UmbracoMasterContentEntity(
+            IPublishedContent content,
+            IEntityIdentityService entityIdentityService)
+        {
+            _content = content;
+            _entityIdentityService = entityIdentityService;
+            Properties = new Dictionary<string, IEnterspeedProperty>
```

Comment:
The master variant doesn't contain any properties, except for at `timestamp` property. Enterspeed doesn't process schemas if the content of the source entity isn't changed, that's why we have the `timestamp` property.